src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <>
+      <div className="PostInfo">
+        <div className="PostInfo__header">
+          <h3 className="PostInfo__title">{title}</h3>
+
+          <p>
+            {' Posted by  '}
+
+            {user && (<UserInfo user={user} />)}
+          </p>
+        </div>
+
+        <p className="PostInfo__body">{body}</p>
+
+        <hr />
+
+        { !comments.length
+          ? <b data-cy="NoCommentsMessage">No comments yet</b>
+          : <CommentList comments={comments} />
```

Comment:
I would do it the nex way

```suggestion
        { comments.length === 0 ? (
          <b data-cy="NoCommentsMessage">No comments yet</b>
        ) : (
          <CommentList comments={comments} />
        )}
```

Reply:
https://prnt.sc/FYFsUTOtjyn4  ))